package.json

```diff
@@ -119,7 +119,12 @@
   },
   "dependencies": {
     "@paulcbetts/system-idle-time": "^1.0.4",
+    "ajv": "^6.2.1",
+    "electron-builder": "^20.0.6",
```

Comment:
does `electron-builder` needs to be in app dependencies? we already have it in dev-dependencies and [documentation] (https://www.electron.build/auto-update) only talkes about `electron-updater`. 

Also: app dependencies needs to go to the `app/package.json` as we use 2 package.json structure. (https://www.electron.build/tutorials/two-package-structure)

Reply:
I added electron-builder for building and publishing my own releases. I will remove it.